doc/adr/0004-tsp-rename-experiment.md

```diff
@@ -0,0 +1,41 @@
+# 3. Tsp rename experiment
```

Comment:
4.

Reply:
I'm aware of other ongoing ADRs and will adjust accordingly